src/lookup.jl

```diff
@@ -95,6 +95,8 @@ findindex(i::Int, r::Base.OneTo{Int}) = 1 <= i <= r.stop ? i : nothing
 
 findindex(i::Int, r::AbstractUnitRange) = first(r) <= i <= last(r) ? 1+Int(i - first(r)) : nothing
 
+findindex(i::AbstractUnitRange{T}, r::AbstractUnitRange{T}) where {T} = findall(∈(i), r)
```

Comment:
Might be clearer just to write intersect? And do we gain anything by demanding the types match?
```suggestion
findindex(i::AbstractUnitRange, r::AbstractUnitRange) = intersect(i, r)
```

Reply:
Not particual reason for that restriction, `intersect` is even better!

Don't see how to accept this change or otherwise edit this PR, so I guess I'll need to create a new one...

Reply:
Surely I can push to the branch, I don't see why but the suggestion button is missing for me too...

Reply:
Trying again:
```suggestion
findindex(i::AbstractUnitRange, r::AbstractUnitRange) = intersect(i, r)
```

Reply:
Ah maybe the branch is deleted at your end? Sorry it took me months to get around to looking at this...

Reply:
Yes, I did delete my repo fork. Thought that github keeps PRs intact anyways...

Reply:
I can also just merge it and then over-write, maybe that's simpler.